workshop/admin.py

```diff
@@ -55,12 +55,20 @@ def get_subscribed_users(self, obj):
     get_joint_secy.short_description = 'Joint Secretary'
     get_subscribed_users.short_description = 'Subscribed Users'
 
+
 @admin.register(Tag)
 class TagAdmin(admin.ModelAdmin):
     list_display = ('tag_name', 'club')
     search_fields = ('tag_name',)
     list_filter = ('club',)
 
+
+@admin.register(WorkshopResource)
+class WorkshopResourceAdmin(admin.ModelAdmin):
+    list_display = ('name', 'link')
+    search_fields = ('name',)
```

Comment:
Can also display the Workshop Name and Resource type, and allow filter by resource type.

Reply:
We'll do it later.